src/main/include/subsystems/Shooter.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <rev/CANSparkMax.h>
+#include <ctre/phoenix/motorcontrol/can/TalonSRX.h>
+#include <units/math.h>
+#include <frc/controller/PIDController.h>
+#include <frc/DigitalInput.h>
+
+#include "Constants.h"
+
+class Shooter
+{
+    public:
+
+    Shooter();
+
+    void runShooter(const double motor_output);
+
+    void runIntake(const double motor_output);
+
+    void moveToAngle(const double angle);
+
+    void periodic();
+
+    void init();
+
+    rev::CANSparkMax m_firing_motorL;
+    rev::CANSparkMax m_firing_motorR;
+    rev::CANSparkMax m_arm_motorT;
+    rev::CANSparkMax m_arm_motorB;
+
+    rev::SparkMaxAbsoluteEncoder m_arm_encoderT;
+    rev::SparkMaxAbsoluteEncoder m_arm_encoderB;
+
+    ctre::phoenix::motorcontrol::can::TalonSRX m_intake;
```

Comment:
With the other members, you named them as motors or encoders or PIDs. Any reason not to do the same here?

```suggestion
    ctre::phoenix::motorcontrol::can::TalonSRX m_intake_motor;
```